src/main/java/core/basesyntax/service/InvalidUserDataException.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.service;
+
+public class InvalidUserDataException extends Exception {
```

Comment:
As you do not plan to handle exception in any way please make it to extend RuntimeException and remove all ```throws``` from methods signatures
```suggestion
public class InvalidUserDataException extends RuntimeException {
```